server/drivers/mysql/index.js

```diff
@@ -45,14 +46,26 @@ function runQuery(query, connection) {
     database: connection.database,
     insecureAuth: connection.mysqlInsecureAuth,
     timezone: 'Z',
-    supportBigNumbers: true
-  });
+    supportBigNumbers: true,
+    ssl: connection.mysqlSsl
+  };
+  // TODO cache key/cert values
+  if (connection.mysqlKey && connection.mysqlCert) {
+    myConfig.ssl = {
+      key: fs.readFileSync(connection.mysqlKey),
+      cert: fs.readFileSync(connection.mysqlCert)
+    };
+    if (connection.mysqlCA) {
+      myConfig.ssl['ca'] = fs.readFileSync(connection.mysqlCA);
+    }
+  }
 
   return new Promise((resolve, reject) => {
+    const connection = mysql.createConnection(myConfig);
```

Comment:
renaming this from `myConnection` to `connection` breaks the whole driver, as the name collides with the argument of the `runQuery` function which has the same name.
This leads to the later check `rows.length < connection.maxRows` *always* being falsy, due to `connection.maxRows` being `undefined`.
=> `rows` will always be empty and `incomplete` will always be true